traffic_control/models/additional_sign.py

```diff
@@ -195,6 +180,22 @@ class AdditionalSignReal(AbstractAdditionalSign):
         blank=True,
         null=True,
     )
+    installation_id = models.CharField(
+        _("Installation id"), max_length=254, blank=True, null=True
+    )
+    installation_details = models.CharField(
+        _("Installation details"), max_length=254, blank=True, null=True
+    )
+    installation_date = models.DateField(_("Installation date"), blank=True, null=True)
+    installation_status = EnumField(
+        InstallationStatus,
+        verbose_name=_("Installation status"),
+        max_length=10,
+        default=InstallationStatus.IN_USE,
+        blank=True,
+        null=True,
+    )
+    installed_by = models.CharField(_("Installed by"), max_length=256, blank=True)
```

Comment:
So what's up here with 254 and then 256 max_length? I feel it really does not matter, but I guess we could stick to one? :D /very minor nitpick

Reply:
The `installation_details` has been added after the other fields which have max length of 256. I don't know what's the reasoning behind 254. I just moved it over as-is. 😄